analysis/classification/utils/db_functions.py

```diff
@@ -164,7 +164,7 @@ def _pull_thread(cv, data_to_label, bq_client, user_langs, method):
             cv.wait()
         elif fetch_job is None:
             cv.release()
-            if method[0] == "Random":
```

Comment:
As above, I think you need to leave this encapsulated in a list, unless you have another solution to make sure the thread picks up the new value if method is changed while the thread is running.